### ISSUE TITLE
Conditional Checks

### ISSUE THREAD

**Issue by howlettt** [koddsson/eslint-plugin-tscompat#700]
Looks like conditional checks aren't supported? For example the following shouldn't generate a warning in browsers that don't support `fetch`:
```
if (fetch) {
  fetch()
}
```

I found the following tests that appear to be checking for this, but when I run them they pass with or without the `if` statement
https://github.com/koddsson/eslint-plugin-tscompat/blob/3710f84a2c6e4e16cf020d3f7ad92027272e6a5f/tests/tscompat.ts#L40-L79
